graphene/types/field.py

```diff
@@ -114,5 +115,24 @@ def __init__(
     def type(self):
         return get_type(self._type)
 
-    def get_resolver(self, parent_resolver):
+    get_resolver = None
+
+    def wrap_resolve(self, parent_resolver):
+        """
+        Wraps a function resolver, using the ObjectType resolve_{FIELD_NAME}
+        (parent_resolver) if the Field definition has no resolver.
+        """
+        if self.get_resolver is not None:
+            warn_deprecation(
```

Comment:
Why the name change?

Reply:
It's much clearer what it does. It wraps the function provided (or defined in self).